roles/setup_gitea/README.md

```diff
@@ -0,0 +1,105 @@
+# setup_gitea role
+
+This role allows the deployment of [Gitea](https://gitea.com/). Gitea is:
+
+[*"a painless, self-hosted, all-in-one software development service. It includes Git hosting, code review, team collaboration, package registry, and CI/CD. It is similar to GitHub, Bitbucket and GitLab."*](https://docs.gitea.com/)
+
+Role tasks:
+  - Validates requirements.
+  - Creates a Gitea deployment.
+  - Creates a route to the Gitea web console.
+  - Initalizes Gitea along with a default user.
+  - Creates the default repository.
+
```

Comment:
- Mirrors an existing repository into the Gitea instance.

Reply:
Thanks for the catch.